application.properties/metasfresh/backend/metasfresh-dist/serverRoot/application.properties

```diff
@@ -18,16 +18,16 @@ spring.rabbitmq.password=guest
 # background: apps server and rabbitmq are often behind the same "gateway" (e.g. docker-host) when seen from the swing-client
 spring.swingui.rabbitmq.host=localhost
 # spring.rabbitmq.host=192.168.99.100  # note: 192.168.99.100 is probably the correct IP if it runs in minikube on your local virtualbox
-spring.swingui.rabbitmq.port=30050
+spring.swingui.rabbitmq.port=5672
```

Comment:
Why did we decide to change all the ports?
ie. k8s: 30050, elk:5672

Is 5672 thge default rabbitmq port? If yes, then I approve of changing the ports. If it's not, then i don't approve of these changes.

Reply:
> Is 5672 thge default rabbitmq port used by everyone else

yes